### DISCUSSION TITLE
Program "openssl" not installed. Please install it

### DISCUSSION THREAD

**Discussion by Danedevz** [pystardust/ani-cli#842] [General] [answered]
I just want to point out the problem I faced when running ani-cli via termux. You might stumble upon this error message:

"Program 'openssl' not installed. Please install it"

despite it's already installed by default on your termux.

So to overcome this problem you might want to install openssl-tool by

```
pkg install openssl-tool
```

So the ani-cli can detect this openssl thingy.

Idk if anyone had solved this problem before. But I already googled this problem and nothing came out. So I hope this can solve your problem. If you have any thoughts or better workaround for this problem feel free to comment below

**Comment by CoolnsX** [comment, marked as answer]
I forgor everytime I do commits.. tehe..

**Reply by Danedevz** [reply]
You might want to add few notes in the installation via termux section about installing openssl-tool package before running the app to help a beginner like me avoid minor errors like this.

Anyway props to you that make this wonderful cli app working like a charm o7
